src/cache/DefaultCache.php

```diff
@@ -39,11 +42,14 @@ public function has(string $name)
    */
     public function set(string $name, $value)
     {
-        if (!$this->cache->has($name)) {
+        if (!in_array($name, $this->element_queue)) {
```

Comment:
Can you explain why this has changed and what `element_queue` is doing exactly?

Reply:
I was getting `Undefined variable $element_queue` , even this is defined as public. That is why, I updated this.

Reply:
That doesn't make much sense to me

Reply:
element_queue keeps tracks of new keys which we wants to store in cache, it helps us to then in manageSize function in which we delete the overflowing keys by deleting them